### PR TITLE
Add watermark recipe

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

File: recipes/watermark/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - certifi
+
+test:
+  imports:
+    - watermark
+    - certifi
+
+about:
+  home: https://github.com/rasbt/watermark
+  license: newBSD
```
Comment (synapticarbors) [inline recipes/watermark/meta.yaml]: Include a license file:

```
license_file: LICENSE
```

**Comment by synapticarbors** [comment]
I'm not a core conda-forge dev, but a few notes based on previous experience: 

- You should include the `license_file` (see inline note in code)
- I think you can get rid of `build.sh` and `bld.bat` if you are using `build: script:`

File: recipes/watermark/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/watermark/meta.yaml]: Normally we do BSD with clauses. So something like `BSD 3-Clause` is preferred.

**Comment by synapticarbors** [comment]
@proinsias my mistake. This has happened before where there is a license file in the github repo, but because of a missing MANIFEST.in file, it doesn't make it onto PyPi. Sorry about that. I'm going to post an issue with the author.

**Comment by synapticarbors** [comment]
I posted an issue (rasbt/watermark#20) so maybe the `LICENSE` file can be added at a later date, possibly after the recipe makes it out of `staged-recipes`.

File: recipes/watermark/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
```
Comment (synapticarbors) [inline recipes/watermark/meta.yaml]: I just installed `watermark` via pip into a conda env on windows and it works, so I wouldn't skip windows in the build unless there is some other compelling reason.

File: recipes/watermark/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - watermark
+    - certifi
+
+about:
+  home: https://github.com/rasbt/watermark
+  license: newBSD
+  license_file: LICENSE
```
Comment (jakirkham) [inline recipes/watermark/meta.yaml]: Please add a comment explaining there is no license file yet with a link to issue ( https://github.com/rasbt/watermark/issues/20 ) and comment this line for now.

File: recipes/watermark/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  imports:
+    - watermark
+    - certifi
+
+about:
+  home: https://github.com/rasbt/watermark
+  license: BSD 3-Clause
+  # license_file: LICENSE # When added to MANIFEST.in – https://github.com/rasbt/watermark/issues/20
+  summary: 'IPython magic function to print date/time stamps andvarious system information.'
+  license_family: BSD
```
Comment (jakirkham) [inline recipes/watermark/meta.yaml]: Please move up with the other license metadata.

File: recipes/watermark/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ipython
+    - certifi
```
Comment (jakirkham) [inline recipes/watermark/meta.yaml]: Please drop `ipython` and `certifi` from build as they should not be needed. Unless I'm missing something.

File: recipes/watermark/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
+{% set name = "watermark" %}
+{% set version = "1.3.1" %}
+{% set md5 = "cf25757d942e3a2ae5077b2ffff172dd" %}
```
Comment (jakirkham) [inline recipes/watermark/meta.yaml]: sha256 would be better.

**Comment by proinsias** [comment]
@jakirkham – need anything more from me?

File: recipes/watermark/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
```
Comment (jakirkham) [inline recipes/watermark/meta.yaml]: What's the reason for skipping on Windows?

File: recipes/watermark/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ipython
```
Comment (jakirkham) [inline recipes/watermark/meta.yaml]: Require due to import of package in `setup.py`. 😒

**Comment by proinsias** [comment]
@jakirkham – I think this is good to go now.

**Comment by jakirkham** [comment]
Agreed. Sorry, the notification got buried.

**Pull request merged**